synapseclient/api/table_services.py

```diff
@@ -69,3 +101,44 @@ async def post_columns(
         column.fill_from_dict(result["list"][i])
 
     return columns
+
+
+async def get_default_columns(
+    view_entity_type: Optional[ViewEntityType] = None,
+    view_type_mask: Optional[ViewTypeMask] = None,
```

Comment:
I think we'll need to make this a type `int`. Saying that the table is going to contain both `Files` and `Folders` is valid for the FileView. ie:

```
FILE = 0x01
FOLDER = 0x08

view_type_mask = FILE | FOLDER
```

Reply:
So then what we would pass to this function is the enum value or an expression like `ViewTypeMask.File.value | ViewTypeMask.Folder.value`?

Reply:
One option is to remove the Enum from these (Only accept an int). This is how https://synapsepythonclient.readthedocs.io/en/latest/reference/experimental/sync/table/#synapseclient.models.Table.query_part_mask was built


Or do it like your suggesting.